### PR TITLE
HW1: ys3593

### PR DESCRIPTION
**Pull request by ys3593**
Modified ys3593.md with links to profile README, project proposal page and project contribution page.

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Project Contribution Description
```
Comment (timkpaine) [inline contribution.md:1]: move to `projects/python/matplotlib.md`

File: proposal.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Project Proposal Description
```
Comment (timkpaine) [inline proposal.md:1]: move to `projects/python/<whatever you want to call it right now>.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: contribution.md (hunk 1 above)
Comment (ys3593) [reply contribution.md]: updated

File: proposal.md (hunk 2 above)
Comment (ys3593) [reply proposal.md]: updated

File: contribution.md (hunk 1 above)
Comment (ys3593) [reply contribution.md]: > move to `projects/python/matplotlib.md`

updated

File: proposal.md (hunk 2 above)
Comment (ys3593) [reply proposal.md]: > move to `projects/python/<whatever you want to call it right now>.md`

updated

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**